recipes/resiprocate/all/conanfile.py

```diff
@@ -0,0 +1,42 @@
+import tempfile
+import os
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+
+
+class ResiprocateConan(ConanFile):
+    name = "resiprocate"
+    description = "The project is dedicated to maintaining a complete, correct, and commercially usable implementation of SIP and a few related protocols. "
+    topics = ("conan", "resiprocate", "sip")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://www.resiprocate.org"
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+    license = "VSL-1.0"
+    install_dir = tempfile.mkdtemp(suffix=name)
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
```

Comment:
This is the usually pattern we have, but it will require you to update other parts too

```suggestion
    @property
    def _source_subfolder(self):
        return "source_subfolder"

    def source(self):
        tools.get(**self.conan_data["sources"][self.version])
        os.rename("{}-{}".format(self.name, self.version), self._source_subfolder)
```

Reply:
Thanks to @uilianries it's fixed!